deploy/kubernetes/dolphinscheduler/Chart.yaml

```diff
@@ -43,10 +43,10 @@ appVersion: dev-SNAPSHOT
 
 dependencies:
 - name: postgresql
-  version: 11.6.6
-  repository: https://charts.bitnami.com/bitnami
+  version: 10.3.18
+  repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
```

Comment:
should we add some comments to desc why we add this change, I think we should at less point to the issue https://github.com/bitnami/charts/issues/10539

Reply:
Yes, I agree with you, or the user may be very confused about why we use this URL.

Reply:
They have a FAQ about this question. Do you think https://github.com/bitnami/charts/issues/10833 is better?

Reply:
Done